### ISSUE TITLE
npm-how-to step 8 returning "author required for publishing" also earlier unclear for setting folder

### ISSUE THREAD

**Issue by theoneinskane** [FreeCodeCamp/FreeCodeCamp#7515]
#### Issue Description
* When running ```npm publish``` receive the following code:

```
theoneinskane:~/workspace/nodey (master) $ npm publish
npm ERR! Linux 4.2.0-c9
npm ERR! argv "/home/ubuntu/.nvm/versions/node/v4.3.2/bin/node" "/home/ubuntu/.nvm/versions/node/v4.3.2/bin/npm" "publish"
npm ERR! node v4.3.2
npm ERR! npm  v3.8.1
npm ERR! code ENEEDAUTH

npm ERR! need auth auth required for publishing
npm ERR! need auth You need to authorize this machine using `npm adduser`

npm ERR! Please include the following file with any support request:
npm ERR!     /home/ubuntu/workspace/nodey/npm-debug.log

```

#### Browser Information
* Safari 9.0.3
* Operating System OS X 10.11.3
* Cloud 9: Version 3.1.1762 (bfa8e706cb)

#### Screenshot
<img width="1440" alt="screen shot 2016-03-12 at 06 49 34" src="https://cloud.githubusercontent.com/assets/8465133/13721186/9ef7902c-e81e-11e5-9c98-648681728ea5.png">

On a more general note, this tutorial is not very clear, points that it doesn't help with are:

- It talks about creating a folder, which I did, but it didn't give much indication why and why not have it in the root workspace folder
- Setting up the user it claims that it doesn't create a real npmjs.org but it does, see my new npmjs user account as evidence: [](https://www.npmjs.com/~nodey1)

**Comment by raisedadead** [comment]
Anyone looking this refer #7080 while addressing this issue.

**Comment by raisedadead** [comment]
Should have been resolved in #8952 , let us know if you still face issues with this
